skeleton-navigation/src/main.ext

```diff
@@ -0,0 +1,8 @@
+// @if browser-ie11
+// Promise polyfill for old browsers
+import 'promise-polyfill/lib/polyfill';
+// @endif
+import Aurelia, { RouterConfiguration/* @if shadow-dom || css-module */, StyleConfiguration }/* @endif */ } from 'aurelia';
+import { MyApp } from './my-app';
+
+Aurelia/* @if shadow-dom */.register(StyleConfiguration.shadowDOM())/* @endif *//* @if css-module */.register(StyleConfiguration.cssModulesProcessor())/* @endif */.register(RouterConfiguration).app(MyApp).start();
```

Comment:
Yes, this file and `src/resource.d.ts` can be pushed down to `common/` folder to avoid duplication.

Reply:
I think main.ext may get nested /* if exp */
please suggest.

Reply:
There is another PR going to be merged too. That will clean up main.ext, then you will only need one condition on `app-with-router`.
Will let you know when it's merged.